client/src/components/UIPatterns/UIPatternsReadMore/UIPatternsReadMore.module.scss

```diff
@@ -0,0 +1,4 @@
+.root {
+  position: relative;
+  max-width: 100%;
+}
```

Comment:
__Curiousity__

What is this doing? I don't care for the purpose of the code review (this is UI patterns, so meh), but removing it has no effect.